dd-java-agent/agent-iast/src/test/groovy/com/datadog/iast/util/ObjectVisitorTest.groovy

```
@@ -74,6 +70,29 @@ class ObjectVisitorTest extends Specification {
     0 * _
   }
 
+  void 'test visiting ignored collection'() {
```

it's not clear to me what we're asserting in this test... I understand that we're not supposed to access the list, so shouldn't we throw an exception in the `get` for instance ? Or count accesses and make sure it's ==0 ?

---

Yes, we could throw an exception but in the end we are asserting that there are no mock invocations of the visitor with `0 * _` which is equivalent.

---

OK thanks I wasn't familiar with that notation